.circleci/config.yml

```diff
@@ -305,6 +305,7 @@ workflows:
           environment: production
           requires:
             - production_deploy_approval
+            - production_deploy
```

Comment:
added this step so that the helm deploy occurs after the kubernetes deploy has completed, this was we ensure that the ingress containing `find-legal-advice.justice.gov.uk` and `laa-fala-production.cloud-platform.service.justice.gov.uk` is deleted before we try to deploy with helm and create an ingress with the same values (which can result in a broken pipeline).

Reply:
Great thanks!